packages/components/src/styles/_popover-component.scss

```
@@ -151,7 +151,7 @@ $popover-gap-inset: calc(-1 * #{$popover-gap-size});
 }
 
 %show-popover {
-	visibility: visible;
+	display: inherit;
```

```suggestion
	display: revert;
```

I think that we would like to revert to the value of the UA styling instead of inheriting a value from another wrapping tag here.